docs/csharp/how-to/compare-strings.md

```diff
@@ -36,7 +36,7 @@ By default, the most common operations:
 
 perform a case-sensitive ordinal comparison and, if necessary, use the current culture. The following example demonstrates that:
 
-[!code-csharp-interactive[Comparing strings using an ordinal comparison](../../../samples/snippets/csharp/how-to/strings/CompareStrings.cs#1)]
+:::code language="csharp-interactive" source="../../../samples/snippets/csharp/how-to/strings/CompareStrings.cs" id="1":::
```

Comment:
I think the id should be `Snippet1` not just `1` ? The build hasn't finished yet, so I'm not sure.

I also think, based on [this](https://docs.microsoft.com/en-us/contribute/code-in-docs#code-snippets-included-by-reference), that the language should be csharp only and set the interactive attribute?

Reply:
`Snippet` is actually optional when referring to a snippet. No matter what ID you create/use the word `Snippet` is always removed before it looks for a snippet.

Reply:
There was build errors caused by omitting the "Snippet" part (for the new syntax only) in previous PRs and was only solved by including the "Snippet" part. Maybe it was a bug that got fixed.